reference/kubectl-plugin.md

```diff
@@ -58,9 +63,9 @@ SUBCOMMANDS:
     uncordon    'Uncordon' resources
 ```
 
-The plugin can be used to get the following resource information:
+You can use the following plugins:
```

Comment:
Seems incorrect as this needs to mention the various resources.

Reply:
> Seems incorrect as this needs to mention the various resources.

Ok, when I checked below, we seem to be clubbing the various options. If that's the intent, then the text can be "you can use the plugin with the following options or in the following way" ?